app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/pickfiles/components/EmailAddressTextField.kt

```diff
@@ -140,26 +140,36 @@ private class EmailAddressTextFieldState(
     fun updateUiTextValue(newValue: TextFieldValue) {
         var hasNewValidRecipientEmail = false
         val lastAddedChar = newValue.text.lastOrNull()
+        if (textFieldValue.text.length + 1 < newValue.text.length) {
+            hasNewValidRecipientEmail = addRecipientAddress(newValue.text)
+        }
 
         if (lastAddedChar == ' ' || lastAddedChar == ',') {
             hasNewValidRecipientEmail = addRecipientAddress()
         }
```

Comment:
Both conditions should be mutually exclusive so we don't call `addRecipientAddress` twice in a row if both conditions are met